docs/core/development/policy/compatibility-policy.rst

```diff
@@ -97,13 +97,13 @@ Procedure for Exceptions to this Policy
 
 **Every change is unique.**
 
-Sometimes, we'll want to make a change that fits with this spirit of this document (keeping Twisted working for applications which rely upon it) but may not fit with the letter of the procedure described above (the change modifies behavior of an existing API sufficiently that something might break).
-Generally, the reason that one would want to do this is to give applications a performance enhancement or bug fix that could break behavior that unanticipated, hypothetical uses of an existing API, but we don't want well-behaved applications to pay the penalty of a deprecation/adopt-a-new-API/removal cycle in order to get the benefits of the improvement if they don't need to.
```

Comment:
I don't this this is more accurate

Reply:
?

Reply:
@wsanchez ?

Reply:
I don't think that `in undesirable` is more accurate than `that unanticipated,`, so I'm not sure that change is appropriate.  "undesirable" is highly subjective, for starters.

Perhaps "unintended" is better?